site/content/en/docs/tasks/run/wrapped_custom_jobs.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Run A Wrapped Custom Job"
+linkTitle: "Custom Job"
+date: 2025-01-14
+weight: 7
+description: >
+  Use an AppWrapper to Run a Custom Job on Kueue.
+---
+
+This page shows how to use [AppWrappers](https://project-codeflare.github.io/appwrapper/) to make
+Kueue's scheduling and resource management capabilities available to Job types that do not have a dedicated
+Kueue integration.  For Jobs that use `PodSpecTemplates` in their definition, this can provide
+a significantly easier approach than [building a custom integration](/docs/tasks/dev/integrate_a_custom_job)
+to enable the use of Kueue with a custom Job type.
+
+This guide is for [batch users](/docs/tasks#batch-user) that have a basic understanding of Kueue. For more information, see [Kueue's overview](/docs/overview).
+
+## Before you begin
+
+1. Make sure you are using Kueue v0.11.0 version or newer and AppWrapper v1.0.0 or newer.
+
+2. Follow the steps in [Run AppWrappers](/docs/tasks/run/appwrappers/#before-you-begin)
+to learn how to enable and configure the `codeflare.dev/appwrapper` integration.
+
+## Example using LeaderWorkerSets as the Custom Job
+
+[LeaderWorkerSets](https://github.com/kubernetes-sigs/lws) are an exanple of a type that
+uses PodSpecTemplates but currently does not have a Kueue integration.
```

Comment:
```suggestion
We use the [LeaderWorkerSets](https://github.com/kubernetes-sigs/lws) to showcase running a workload 
of a custom type.
```
We are likely to have the integration supported in 0.11 (but disabled by default), so maybe we can relax the statement.

Reply:
Rewritten.